CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [_Unreleased_](https://github.com/freckle/haskell-library-template/compare/v__...main)
+
+- Unrelease features
```

Comment:
```suggestion
- Unreleased features
```

FWIW though, I've started leaving this block empty. As folks land things in `main`, I find we rarely remember to also pre-write CHANGELOG entries in this section for future use. I always click the compare link above to figure out what to write when actually releasing -- and I don't mind that at all, to reduce the burden on the actual work itself.

Reply:
This was mostly so I could avoid `restyled` yelling about extra whitespace and provide some examples.